README.md

```diff
@@ -0,0 +1,24 @@
+# valid-github-username
+
+This node.js program checks if a username is a valid github username. An example output from this program is:
```

Comment:
this is not the output of the program, but this is really useful .  Put it in a section `## Checks it performs `

The first section should be 'Installation'

Then , if possible -- i.e. optional, a section "Usage" , which shows how to use it in Node code.

Reply:
Okay, will do. For the Usage section is it just `npm test`?